eZ/Publish/Core/REST/Common/FieldTypeProcessor/StringProcessor.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * File containing the TextLineProcessor class.
```

Comment:
Thanks @crevillo.

I started on the TextLine but the processor is based on the returned identifier by getFieldTypeIdentifier.